jobs/pull-test-infra-bazel.sh

```diff
@@ -20,6 +20,8 @@ set -o pipefail
 # Cache location.
 export TEST_TMPDIR="/root/.cache/bazel"
 
+find -L bazel-testlogs -name 'test.xml' -type f -exec rm '{}' +
```

Comment:
Done. I also had to put it after the build step, since the symlink won't exist in a fresh clone.